client/channel.go

```diff
@@ -41,12 +42,40 @@ func (c *Client) ListChannels(appID string) ([]types.Channel, error) {
 	return nil, errors.New("unknown app type")
 }
 
-func (c *Client) GetChannel(appID string, channelID string) (interface{}, interface{}, error) {
-	return nil, nil, nil
+func (c *Client) GetChannel(appID string, channelID string) (*channels.AppChannel, []channels.ChannelRelease, error) {
+	appType, err := c.GetAppType(appID)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if appType == "platform" {
+		return c.PlatformClient.GetChannel(appID, channelID)
+	} else if appType == "ship" {
+		return c.ShipClient.GetChannel(appID, channelID)
+	} else if appType == "kots" {
+		return c.KotsClient.GetChannel(appID, channelID)
+	}
+	return nil, nil, errors.New("unknown app type")
+
 }
 
 func (c *Client) ArchiveChannel(appID string, channelID string) error {
-	return nil
+	appType, err := c.GetAppType(appID)
+	if err != nil {
+		return err
+	}
+
+	if appType == "platform" {
+		return c.PlatformClient.ArchiveChannel(appID, channelID)
+	} else if appType == "ship" {
+		// return c.ShipClient.ArchiveChannel(appID, channelID)
+		return nil
```

Comment:
do you think if this finishes without an error, that the user will believe that the channel was archived?

Reply:
good point - will add that to the 'sorry, come back later' list.